Bottom: 67eec87fa9af69af71c0117c513e627fc1ccd969
Top:    2d92ae89cbff3b1eccae9de8b9718649004e2453
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-14 16:27:19 -0700

Refresh of dbg-out

---

diff --git a/src/preload.c b/src/preload.c
index 8770161..efdb414 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -1076,6 +1076,7 @@ int __fxstat64(int ver, int socket, struct stat64 *buf)
 {
 //	int fd, ret;
 
+	fprintf(fout, "%s\n", __func__); fflush(fout);
 //	if (fd_get(socket, &fd) == fd_rsocket) {
 //		ret = real.fxstat64(ver, socket, buf);
 //		if (!ret)
@@ -1089,5 +1090,6 @@ int __fxstat64(int ver, int socket, struct stat64 *buf)
 
 int __fxstat(int ver, int socket, struct stat *buf)
 {
+	fprintf(fout, "%s\n", __func__); fflush(fout);
 	return real.fxstat(ver, socket, buf);
 }
